# src_luaconf.h.dpatch by John V. Belmonte <jbelmonte@debian.org>
# Set Lua's default PATH and CPATH.
# See http://lua-users.org/wiki/LuaPowerPatches
# URL: http://lua-users.org/files/wiki_insecure/power_patches/5.1/module_paths.patch
diff -urNad lua-5.1.5-orig/src/luaconf.h lua-5.1.5/src/luaconf.h
--- lua-5.1.5-orig/src/luaconf.h	2008-02-11 18:25:08.000000000 +0200
+++ lua-5.1.5/src/luaconf.h	2015-05-02 12:12:27.000000000 +0300
@@ -95,13 +95,17 @@
 
 #else
 #define LUA_ROOT	"/usr/local/"
+#define LUA_ROOT2	"/usr/"
 #define LUA_LDIR	LUA_ROOT "share/lua/5.1/"
+#define LUA_LDIR2	LUA_ROOT2 "share/lua/5.1/"
 #define LUA_CDIR	LUA_ROOT "lib/lua/5.1/"
+#define LUA_CDIR2	LUA_ROOT2 "lib/lua/5.1/"
 #define LUA_PATH_DEFAULT  \
 		"./?.lua;"  LUA_LDIR"?.lua;"  LUA_LDIR"?/init.lua;" \
-		            LUA_CDIR"?.lua;"  LUA_CDIR"?/init.lua"
+		            LUA_CDIR"?.lua;"  LUA_CDIR"?/init.lua;" \
+		            LUA_LDIR2"?.lua;"  LUA_LDIR2"?/init.lua"
 #define LUA_CPATH_DEFAULT \
-	"./?.so;"  LUA_CDIR"?.so;" LUA_CDIR"loadall.so"
+	"./?.so;" LUA_CDIR"?.so;" LUA_CDIR2"?.so;" LUA_CDIR"loadall.so"
 #endif
 
 
